.circleci/config.yml

```diff
@@ -25,7 +35,6 @@ jobs:
       - run: echo $MYSQL_TEST
       - run: curl -L https://github.com/coreos/etcd/releases/download/v3.1.7/etcd-v3.1.7-linux-amd64.tar.gz -o etcd-v3.1.7-linux-amd64.tar.gz
       - run: tar xzvf etcd-v3.1.7-linux-amd64.tar.gz
-      - run: sudo apt-get install mysql-client
```

Comment:
Modified `run_tests.sh` instead. Now, mysql client is no longer required for CircleCI env.